mongodb/spring-cloud-starter-stream-source-mongodb/src/main/java/org/springframework/cloud/stream/app/mongodb/source/MongoDBProperties.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2016 the original author or authors.
+ * Licensed under the Apache License, Version 2.0 (the "License") ;
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *          http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.stream.app.mongodb.source;
+
+import org.hibernate.validator.constraints.NotBlank;
+import org.hibernate.validator.constraints.NotEmpty;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * @author Adam Zwickey
+ *
+ */
+@ConfigurationProperties
+public class MongoDBProperties {
+
+    private String _uri, _collection, _exp;
+    private long _pollingRate = 10000;
```

Comment:
That's not our code style to use underscore. More over all properties must be declared separately. 
Please, pay attention that we now is fully based on the Spring Boot, so, `MongoAutoConfiguration` and its `MongoProperties` must be in count.
Therefore here we must have only those properties which are related to `Source` application: `collection`, `query`.
BTW, it would be better to rename this class to reflect that it is for `Source`.